nuxeo-error.html

```diff
@@ -0,0 +1,126 @@
+<!--
+(C) Copyright 2017 Nuxeo SA (http://nuxeo.com/) and others.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="nuxeo-i18n-behavior.html">
+<!--
+An element to display errors.
+
+Example:
+
+    <nuxeo-error code="404"></nuxeo-error>
+
+@group Nuxeo UI
+@element nuxeo-error
+-->
+<dom-module id="nuxeo-error">
+  <template>
+    <style>
+       :host {
+        display: block;
+        margin: 16px;
```

Comment:
margin was in a `main` container which I have since removed, should I just remove the margin ?